docs/en/compute/Compute-HBase.md

```diff
@@ -14,15 +14,18 @@ that you can easily store HBase tables into Alluxio at various storage levels.
 
 ## Prerequisites
 
-The prerequisite for this part is that you have
-[Java](Java-Setup.html). Your Alluxio cluster should also be
-set up in accordance to these guides for either [Local Mode](Running-Alluxio-Locally.html) or
-[Cluster Mode](Running-Alluxio-on-a-Cluster.html).
-
-Please follow the guides for setting up HBase on
-[Apache HBase Configuration](https://hbase.apache.org/book.html#configuration).
-
-## Configuration
+* [Setup Java](Java-Setup.html) for Java 8 Update 60 or higher (8u60+), 64-bit.
+* An Alluxio cluster has been set up and is running according to either 
+[Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
```

Comment:
new way to specify the link 
`[link text]({{ site.baseurl }}{% link en/deploy/Running-Alluxio-Locally.md %})`

Reply:
@apc999 have the `en/Java-Setup.md` already been removed?
Will change the links, thanks

Reply:
no need to hyper link Jave setup any more. it is out of the scope of this documentation

Reply:
Removed Java setup, and changed the links